components/raftstore-v2/src/batch/store.rs

```diff
@@ -345,6 +349,7 @@ where
             self_disk_usage: DiskUsage::Normal,
             snap_mgr: self.snap_mgr.clone(),
             pd_scheduler: self.pd_scheduler.clone(),
+            tablet_gc_scheduler: self.tablet_gc_scheduler.clone(),
```

Comment:
Should put inside `Schedulers`.

Reply:
I'll make it a simple thread instead if there's no task that needs scheduling.

Reply:
There is. For example, we need it to cleanup split tablet.